src/main/java/uk/gov/hmcts/cmc/claimstore/config/properties/notifications/EmailTemplates.java

```diff
@@ -63,6 +63,9 @@
     @NotEmpty
     private String offerCounterSignedEmailToOtherParty;
 
+    @NotEmpty
+    private String claimantResponseWithMediationIssued;
```

Comment:
Can we group notification together so template without mediation and with mediation are kept together.

Also applying the same naming convention will help.